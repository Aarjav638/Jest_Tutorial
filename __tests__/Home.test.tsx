import {render} from '@testing-library/react-native';
import Home from '../screens/home';
import React from 'react';
test('renders the UI correctly', () => {
  const tree = render(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
