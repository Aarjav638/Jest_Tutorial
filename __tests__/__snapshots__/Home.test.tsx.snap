// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI correctly 1`] = `
<View>
  <RCTScrollView
    data={
      [
        undefined,
        undefined,
        undefined,
        undefined,
        undefined,
        undefined,
        undefined,
        undefined,
        undefined,
        undefined,
      ]
    }
    getItem={[Function]}
    getItemCount={[Function]}
    keyExtractor={[Function]}
    onContentSizeChange={[Function]}
    onLayout={[Function]}
    onMomentumScrollBegin={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={0.0001}
    stickyHeaderIndices={[]}
    viewabilityConfigCallbackPairs={[]}
  >
    <View>
      <View
        onFocusCapture={[Function]}
        onLayout={[Function]}
        style={null}
      >
        <View
          style={
            {
              "flex": 1,
              "flexDirection": "column",
              "gap": 10,
              "height": "auto",
              "justifyContent": "center",
              "marginBottom": 10,
            }
          }
        >
          <View
            style={
              {
                "alignItems": "center",
                "columnGap": 10,
                "flexDirection": "row",
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "10%",
                }
              }
            />
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "80%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "backgroundColor": "grey",
                "padding": 100,
                "width": "60%",
              }
            }
          />
        </View>
      </View>
      <View
        onFocusCapture={[Function]}
        onLayout={[Function]}
        style={null}
      >
        <View
          style={
            {
              "flex": 1,
              "flexDirection": "column",
              "gap": 10,
              "height": "auto",
              "justifyContent": "center",
              "marginBottom": 10,
            }
          }
        >
          <View
            style={
              {
                "alignItems": "center",
                "columnGap": 10,
                "flexDirection": "row",
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "10%",
                }
              }
            />
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "80%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "backgroundColor": "grey",
                "padding": 100,
                "width": "60%",
              }
            }
          />
        </View>
      </View>
      <View
        onFocusCapture={[Function]}
        onLayout={[Function]}
        style={null}
      >
        <View
          style={
            {
              "flex": 1,
              "flexDirection": "column",
              "gap": 10,
              "height": "auto",
              "justifyContent": "center",
              "marginBottom": 10,
            }
          }
        >
          <View
            style={
              {
                "alignItems": "center",
                "columnGap": 10,
                "flexDirection": "row",
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "10%",
                }
              }
            />
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "80%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "backgroundColor": "grey",
                "padding": 100,
                "width": "60%",
              }
            }
          />
        </View>
      </View>
      <View
        onFocusCapture={[Function]}
        onLayout={[Function]}
        style={null}
      >
        <View
          style={
            {
              "flex": 1,
              "flexDirection": "column",
              "gap": 10,
              "height": "auto",
              "justifyContent": "center",
              "marginBottom": 10,
            }
          }
        >
          <View
            style={
              {
                "alignItems": "center",
                "columnGap": 10,
                "flexDirection": "row",
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "10%",
                }
              }
            />
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "80%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "backgroundColor": "grey",
                "padding": 100,
                "width": "60%",
              }
            }
          />
        </View>
      </View>
      <View
        onFocusCapture={[Function]}
        onLayout={[Function]}
        style={null}
      >
        <View
          style={
            {
              "flex": 1,
              "flexDirection": "column",
              "gap": 10,
              "height": "auto",
              "justifyContent": "center",
              "marginBottom": 10,
            }
          }
        >
          <View
            style={
              {
                "alignItems": "center",
                "columnGap": 10,
                "flexDirection": "row",
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "10%",
                }
              }
            />
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "80%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "backgroundColor": "grey",
                "padding": 100,
                "width": "60%",
              }
            }
          />
        </View>
      </View>
      <View
        onFocusCapture={[Function]}
        onLayout={[Function]}
        style={null}
      >
        <View
          style={
            {
              "flex": 1,
              "flexDirection": "column",
              "gap": 10,
              "height": "auto",
              "justifyContent": "center",
              "marginBottom": 10,
            }
          }
        >
          <View
            style={
              {
                "alignItems": "center",
                "columnGap": 10,
                "flexDirection": "row",
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "10%",
                }
              }
            />
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "80%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "backgroundColor": "grey",
                "padding": 100,
                "width": "60%",
              }
            }
          />
        </View>
      </View>
      <View
        onFocusCapture={[Function]}
        onLayout={[Function]}
        style={null}
      >
        <View
          style={
            {
              "flex": 1,
              "flexDirection": "column",
              "gap": 10,
              "height": "auto",
              "justifyContent": "center",
              "marginBottom": 10,
            }
          }
        >
          <View
            style={
              {
                "alignItems": "center",
                "columnGap": 10,
                "flexDirection": "row",
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "10%",
                }
              }
            />
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "80%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "backgroundColor": "grey",
                "padding": 100,
                "width": "60%",
              }
            }
          />
        </View>
      </View>
      <View
        onFocusCapture={[Function]}
        onLayout={[Function]}
        style={null}
      >
        <View
          style={
            {
              "flex": 1,
              "flexDirection": "column",
              "gap": 10,
              "height": "auto",
              "justifyContent": "center",
              "marginBottom": 10,
            }
          }
        >
          <View
            style={
              {
                "alignItems": "center",
                "columnGap": 10,
                "flexDirection": "row",
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "10%",
                }
              }
            />
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "80%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "backgroundColor": "grey",
                "padding": 100,
                "width": "60%",
              }
            }
          />
        </View>
      </View>
      <View
        onFocusCapture={[Function]}
        onLayout={[Function]}
        style={null}
      >
        <View
          style={
            {
              "flex": 1,
              "flexDirection": "column",
              "gap": 10,
              "height": "auto",
              "justifyContent": "center",
              "marginBottom": 10,
            }
          }
        >
          <View
            style={
              {
                "alignItems": "center",
                "columnGap": 10,
                "flexDirection": "row",
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "10%",
                }
              }
            />
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "80%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "backgroundColor": "grey",
                "padding": 100,
                "width": "60%",
              }
            }
          />
        </View>
      </View>
      <View
        onFocusCapture={[Function]}
        onLayout={[Function]}
        style={null}
      >
        <View
          style={
            {
              "flex": 1,
              "flexDirection": "column",
              "gap": 10,
              "height": "auto",
              "justifyContent": "center",
              "marginBottom": 10,
            }
          }
        >
          <View
            style={
              {
                "alignItems": "center",
                "columnGap": 10,
                "flexDirection": "row",
              }
            }
          >
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "10%",
                }
              }
            />
            <View
              style={
                {
                  "backgroundColor": "grey",
                  "padding": 8,
                  "width": "80%",
                }
              }
            />
          </View>
          <View
            style={
              {
                "backgroundColor": "grey",
                "padding": 100,
                "width": "60%",
              }
            }
          />
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
